easybuild/easyconfigs/s/SlurmViewer/SlurmViewer-1.0.2-GCCcore-13.3.0.eb

```
@@ -0,0 +1,61 @@
+# #
+# Authors:    
+#  - Patrick de Koning <p.j.h.de_koning@lumc.nl>
+#  - Prerak Mody <prerakmody@gmail.com>
+# #
+easyblock = 'PythonBundle'
+
+name = 'SlurmViewer'
+version = '1.0.2'
+
+homepage = 'https://gitlab.com/lkeb/slurm_viewer'
+description = """One-stop command to view the status of a Slurm cluster, including nodes, jobs and resource utilization"""
```

CI is failing because this line is a little bit too long:

> SlurmViewer-1.0.2-GCCcore-13.3.0.eb:12:121: E501 line too long (122 > 120 characters)

Could you split the line?

```suggestion
description = """One-stop command to view the status of a Slurm cluster,
including nodes, jobs and resource utilization."""
```